sdk/core/azure-core/src/main/java/com/azure/core/util/UserAgentUtil.java

```diff
@@ -0,0 +1,118 @@
+package com.azure.core.util;
+
+/**
+ * Utility for building user agent string for Azure client libraries as specified in the
+ * <a href="https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy">design guidelines</a>.
+ */
+public final class UserAgentUtil {
```

Comment:
Does this utility class resolve this issue? https://github.com/Azure/azure-sdk-for-java/issues/7549

Reply:
Yes, I had not seen this issue earlier but I think this PR will address this issue too.